### PR TITLE
Add Mobile Menu to Website

### PR DESCRIPTION
**Pull request by mmatongo**
## This pull request adds the following changes.
- Adds JavaScript to enable mobile menu.
- Makes menu items functional.

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-use-before-define */
+// eslint-disable-next-line func-names
```
Comment (Shaher-11) [inline js/menu.js:2]: Kindly remove the eslint disable

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @mmatongo,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comment below:point_down:

## Optional Suggestions💡
_The following point is highly recommended to improve your code quality, Please consider implementing it.

- Kindly increase the space between each one of the menu items.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: js/menu.js (hunk 1 above)
Comment (mmatongo) [reply js/menu.js:2]: Could you offer some advice on this?

I can easily get rid of the 
`// eslint-disable-next-line func-names`

But if I try to remove the 
`/* eslint-disable no-use-before-define */`

I get the following errors
```cmd
6:45  error    'removeShow' was used before it was defined  no-use-before-define
10:42  error    'addShow' was used before it was defined     no-use-before-define
```

Comment (Shaher-11) [reply js/menu.js:2]: `removeShow` and `addShow` both were used then defined so define them before the usage

File: index.html [hunk 2]
```diff
@@ -10,36 +10,39 @@
...
+    function gtag() { dataLayer.push(arguments); }
     gtag('js', new Date());
     gtag('config', 'UA-123825544-3');
   </script>
+  <script defer src="js/menu.js"></script>
   <title>Daniel's Portfolio</title>
 </head>
+
 <body>
   <header class="toolbar">
```
Comment (rloterh) [inline index.html:22]: - [ ] OPTIONAL
Your mobile menu items do not have enough vertical spacing between the items as can be seen in the provided  [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2228&scaling=min-zoom&page-id=1%3A468) design. We actually our design to as much as seemingly closely similar to mobile menu on the chosen design. Kindly ensure that the font color, margins and paddings closely matches with the provided design.

**Your design:**
image

**The original design**
image

File: js/menu.js (hunk 1 above)
Comment (rloterh) [reply js/menu.js:2]: Well resolved! ✔️

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Daniel and Henry, 

Great job for making the necessary changes recommended by the previous reviewer. However, there's always room for improvement.
Kindly consider my OPTIONAL comments

## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

- _Nice to add the the details of the the second author `Henry` in the Authors' section of your ReadMe._

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**

<br>
Your project is complete! Congratulations! 🥂There is nothing else to say other than... it's time to merge it 👍.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**